app/src/main/java/org/mozilla/fenix/library/bookmarks/selectfolder/SelectBookmarkFolderFragment.kt

```diff
@@ -101,11 +99,9 @@ class SelectBookmarkFolderFragment : Fragment(), AccountObserver {
     }
 
     override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
-        val visitedAddBookmark = SelectBookmarkFolderFragmentArgs.fromBundle(arguments!!).visitedAddBookmark
-        if (!visitedAddBookmark) {
+        val args: SelectBookmarkFolderFragmentArgs by navArgs()
+        if (!args.visitedAddBookmark) {
             inflater.inflate(R.menu.bookmarks_select_folder, menu)
-            menu.findItem(R.id.add_folder_button).icon.colorFilter =
-                PorterDuffColorFilter(context!!.getColorFromAttr(R.attr.primaryText), SRC_IN)
```

Comment:
I see you remove the primaryText here for `add_folder_button` but I don't see an xml modified?

Reply:
The `iconTint` is already set in bookmarks_select_folder.xml